fe/fe-core/src/main/java/com/starrocks/binlog/BinlogConfig.java

```diff
@@ -0,0 +1,96 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.binlog;
+
+import com.starrocks.common.io.Writable;
+import com.starrocks.thrift.TBinlogConfig;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+public class BinlogConfig implements Writable {
+    private Long version;
+    private boolean binlogEnable;
+    private Long binlogTtlSecond;
+    private Long binlogMaxSize;
+
+    public BinlogConfig(Long version, boolean binlogEnable, Long binlogTtlSecond, Long binlogMaxSize) {
+        this.version = version;
+        this.binlogEnable = binlogEnable;
+        this.binlogMaxSize = binlogMaxSize;
+    }
+
+    public BinlogConfig(BinlogConfig binlogConfig) {
+        this.version = binlogConfig.getVersion();
+        this.binlogTtlSecond = binlogConfig.getBinlogTtlSecond();
+        this.binlogMaxSize = binlogConfig.getBinlogMmaxSize();
+    }
+
+    public BinlogConfig(boolean binlogEnable, Long binlogTtlSecond, Long binlogMaxSize) {
+        this(-1L, binlogEnable, binlogTtlSecond, binlogMaxSize);
+    }
+    public BinlogConfig() {}
+
+    public Long getVersion() {
+        return version;
+    }
+
+    public void setVersion(Long version) {
+        this.version = version;
+    }
+
+    public boolean getBinlogEnable() {
+        return binlogEnable;
+    }
+
+    public void setBinlogEnable(boolean binlogEnable) {
+        this.binlogEnable = binlogEnable;
+    }
+
+    public Long getBinlogTtlSecond() {
+        return binlogTtlSecond;
+    }
+
+    public void setBinlogTtlSecond(Long binlogTtlSecond) {
+        this.binlogTtlSecond = binlogTtlSecond;
+    }
+
+    public Long getBinlogMmaxSize() {
+        return binlogMaxSize;
+    }
+
+    public void setBinlogMaxSize(Long binlogMaxSize) {
+        this.binlogMaxSize = binlogMaxSize;
+    }
+
+    public void incVersion() {
+        version++;
+    }
+
+
+    public TBinlogConfig toTBinlogConfig() {
+        TBinlogConfig tBinlogConfig = new TBinlogConfig();
+        tBinlogConfig.setVersion(version);
+        tBinlogConfig.setBinlog_enable(binlogEnable);
+        tBinlogConfig.setBinlog_ttl_second(binlogTtlSecond);
+        tBinlogConfig.setBinlog_max_size(binlogMaxSize);
+        return tBinlogConfig;
+    }
+
+    @Override
+    public void write(DataOutput out) throws IOException {
+        out.writeLong(version);
```

Comment:
Please serialize it into a JSON string to make it version compatible, otherwise we could not add field in the future